lib/middleware/token.js

```diff
@@ -52,15 +54,15 @@ var TokenError = require('../errors/tokenerror');
  */
 module.exports = function token(server, options) {
   options = options || {};
-  
-  if (!server) { throw new TypeError('oauth2orize.token middleware requires a server argument'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.token}} middleware requires a server argument')); }
```

Comment:
I am also curious about how the "translation" work.
Let's say if we wrap {{server}},  the translated result could be 
`nécessite un 'server' d' argument`
but also could be 
`nécessite un argument de server`
.
It depends on how "smart" the translator is, and also depends on do we have people manually correct the translated strings to make sure them making sense in each languague.